src/gwt/acesupport/acemode/r_scope_tree.js

```diff
@@ -512,6 +513,25 @@ define('mode/r_scope_tree', ["require", "exports", "module"], function(require,
          this.printScopeTree();
       };
 
+      this.onTestScopeStart = function(desc, startPos, scopePos) {
+         debuglog("adding test_that() brace-scope " + desc);
+         var name = "test_that(" + desc + ")";
+         this.$root.addNode(
+            new this.$ScopeNodeFactory(
+               "test_that(" + desc + ")",
+               scopePos,
+               startPos,
+               ScopeNode.TYPE_BRACE,
+               {
+                  "name": "✅ " + desc,
```

Comment:
IMHO we should just use the name as-is, and consider an icon or styling (if any) on the GWT side.

(In addition, some fonts have trouble rendering certain UTF-8 symbols, so if we do want an icon or embellishment I think we should use an image)

It might also suffice to just use italics for the text label, or something similar, or leave the quotes in (since only test names would be quoted)